### DISCUSSION TITLE
Apply `allowedVersion` to every dependency/package

### DISCUSSION THREAD

**Discussion by ahatzz11** [renovatebot/renovate#18547] [Request Help] [answered]
👋  We are running renovate against our organization, and I'm looking at making it so only certain types of versions are suggested by renovated, specifically standard semver, e.g. `1.2.3`. 

I'd like to apply this rule to every package/dependency, how would I define this in my config?

I haven't tested this but is regex allowed in `matchPackageNames`? Can I leave that field blank to just have a single `allowedVersions` config applied?

```json
{
  "packageRules": [
    {
      "matchPackageNames": ["*"],
      "allowedVersions": "/^([0-9]+.[0-9]+.[0-9]+$/"
    }
}
```

Thanks in advance!

**Comment by rarkins** [comment, marked as answer]
Use `matchPackagePatterns`

**Reply by ahatzz11** [reply]
@rarkins 🤦 I can't believe I didn't see that option looking at the docs, thanks for the super quick answer!

**Reply by ahatzz11** [reply]
@rarkins quick follow-up - if I apply this rule first and then have more specific rules (like a single version for a specific package) after it, does 'latest rule win'?

**Reply by rarkins** [reply]
Yes, latest rule wins. So you'd need to set `allowedVersions` to `null`. BTW `matchCurrentVersion` can be helpful to use too
